engine/client/drivers/cloud.go

```diff
@@ -0,0 +1,99 @@
+package drivers
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"net"
+	"net/url"
+	"time"
+
+	"github.com/dagger/dagger/internal/cloud"
+	"github.com/dagger/dagger/internal/cloud/auth"
+)
+
+var (
+	TLSHandshakeTimeout = 15 * time.Second
+)
+
+func init() {
+	register("dagger-cloud", &daggerCloudDriver{})
```

Comment:
nit: could we just claim `cloud://` instead?

Reply:
Just followed existing conventions here:
- `docker-container`
- `docker-image`
- `kube-pod`
- `podman-container`

`tcp`, `unix` & `ssh` are the only exceptions which are standard protocols.

Don't you think that introducing `cloud://` here would be confusing?